addons/stock/report/product_templates.xml

```diff
@@ -12,9 +12,9 @@
 ^XA^CI28
                             <t t-set="price_unformatted" t-value="pricelist._get_product_price(product, 1)"/>
                             <t t-set="price" t-value="currency_id.format(pricelist._get_product_price(product, 1))"/>
-                            <t t-set="reference_unit_name" t-value="product.uom_id.category_id.uom_ids.filtered(lambda u: u.uom_type == 'reference').name"/>
-                            <t t-set="price_per_unit" t-value="currency_id.format(price_unformatted / product.uom_id.factor_inv)"/>
-                            <t t-set="show_price_per_unit" t-value="product.uom_id.factor_inv != 1"/>
+                            <t t-set="reference_unit_name" t-value="product.uom_id.relative_uom_id.name"/>
```

Comment:
Note that the point of this was to be able to display a "Price pet Unit", as the reference unit *always* had a factor of 1.
Since that's no longer the case, we can't just slap the relative_uom_id and be done with it. This won't work if you have a uom chain longer that 2, or if your relative_uom_id has a factor != 1.